src/prelude.rs

```diff
@@ -1,9 +1,11 @@
-//! Prelude for openmls.
-//! Include this to get access to all the public functions of openmls.
+//! Prelude for OpenMLS.
```

Comment:
We have to clean up the prelude (to only contain the good and necessary stuff) and API (all the public functions). I'll do that after doing the errors.